src/query/time_to_live/time_to_live.cpp

```diff
@@ -70,17 +70,18 @@ void TTL::Setup_(TDbAccess db_acc, InterpreterContext *interpreter_context) {
   auto TTL = [interpreter = std::move(interpreter)]() {
     memgraph::query::DiscardValueResultStream result_stream;
     bool finished = false;
+    // Using microseconds to be aligned with timestamp() query, could just use seconds
+    const auto now = std::chrono::system_clock::now();
+    const auto now_us = std::chrono::duration_cast<std::chrono::microseconds>(now.time_since_epoch());
+    spdlog::trace("Running TTL at {}", now);
     while (!finished) {
-      // Using microseconds to be aligned with timestamp() query, could just use seconds
-      const auto now =
-          std::chrono::duration_cast<std::chrono::microseconds>(std::chrono::system_clock::now().time_since_epoch());
       try {
         interpreter->BeginTransaction();
         auto prepare_result =
             interpreter->Prepare("MATCH (n:TTL) WHERE n.ttl < $now WITH n LIMIT $batch DETACH DELETE n;",
-                                 [now](auto) {
+                                 [now_us](auto) {
```

Comment:
It's just a variable name change. 
I am using the `now` variable to store system time and `now_us` to the duration since epoch.
I am reusing system time to have better logs (using it at the start end end of the loop).